.changeset/tiny-pigs-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ag.ds-next/badge': minor
```

Comment:
Removing a feature is a breaking change right? I think this should probably be a major.